### PR TITLE
[v4.16] pull ose-operator-registry-rhel9 image from brew.registry.redhat.io

### PR DESCRIPTION
**Pull request by tiraboschi**

### CODE REVIEW

**Comment by tiraboschi** [comment]
Not working:
```
Trying to pull brew.registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.16...
Error: initializing source docker://brew.registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.16: unable to retrieve auth token: invalid username/password: unauthorized: Please login to the Red Hat Registry using your Customer Portal credentials. Further instructions can be found here: https://access.redhat.com/RegistryAuthentication
```

File: v4.16/catalog.Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # The base image is expected to contain
 # /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
-# TODO: bump to v4.16 ASAP!!!
-FROM registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.15
+# TODO: remove `brew.` prefix ASAP!!!
+FROM brew.registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.16
```
Comment (orenc1) [inline v4.16/catalog.Dockerfile:4]: please update to:
`brew.registry.redhat.io/rh-osbs/openshift-ose-operator-registry-rhel9:v4.16`

**Comment by orenc1** [comment]
please update the pullspec as advised

**Pull request closed without merging**